lustre_exporter_test.go

```diff
@@ -1631,7 +1644,7 @@ func TestCollector(t *testing.T) {
 		{"lustre_rpcs_in_flight", "Current number of RPCs that are processing during the snapshot.", gauge, []labelPair{{"component", "client"}, {"operation", "write"}, {"size", "9"}, {"target", "lustrefs-OST0000-osc-ffff88105db50000"}, {"type", "osc"}}, 272560, false},
 
 		// Generic Metrics
-		{"lustre_health_check", "Current health status for the indicated instance: 1 refers to 'healthy', 0 refers to 'unhealthy'", gauge, []labelPair{{"component", "generic"}, {"target", "lustre"}}, 1, false},
+		{"lustre_health_check", "Current health status for the indicated instance: 1 refers to 'healthy', 0 refers to 'unhealthy'", gauge, []labelPair{{"component", "health"}, {"target", "lustre"}}, 1, false},
```

Comment:
Teeeeechnically I think this should be in a different section (`//Health Metrics`), but I'm definitely not offended by this.

Reply:
I'll move it! Thanks for catching.